src/main/java/org/ays/emergency_application/model/request/EmergencyEvacuationApplicationRequest.java

```diff
@@ -76,22 +74,22 @@ public class EmergencyEvacuationApplicationRequest {
     @Valid
     private AysPhoneNumberRequest applicantPhoneNumber;
 
-
     @JsonIgnore
```

Comment:
@egehanasal 'ın söylediklerine ek olarak burayı da eski hâline alabiliriz. Üstünde bir boşluk vardı bu silinmiş
```suggestion

    @JsonIgnore
```